config.tpl.yml

```diff
@@ -9,10 +9,10 @@ use_existing_rg: false
 tags:
   env: dev
   project: azhop
-# Define an ANF account, single pool and volume
-# If not present assume that there is an existing NFS share for the users home directory
+# Define an Azure Netapp Files (ANF) account, single pool and volume
+# If not present, assume that there is an existing NFS share for the users home directory
 anf:
-  # Size of the ANF pool and unique volume
+  # Size of the ANF pool and unique volume (min: 4TB, max: 500TB)
```

Comment:
Maximum size is 100TB as this is a volume and not 500TB. If you manually create an ANF outside of azhop, you can still use it, you need to specify the mount values in the config file.